patch_recurrence/config/constants.yml

```diff
@@ -1,6 +1,6 @@
 K_NEAREST: 9
 PATCH_SIZE: 7
-PAIR_THRESHOLD: 0.8
+PAIR_THRESHOLD: 0.7
```

Comment:
I thought the threshold was too strict because initially I wasn't getting too many pairs. I will see if this has changed now and if so change it back to 0.8